From b18923b970a39a6ae83a7e18ceae10d3bc66576b Mon Sep 17 00:00:00 2001
From: Orson Zhai <orson.zhai@linaro.org>
Date: Wed, 28 Nov 2018 13:00:06 +0800
Subject: [PATCH 2/3] ddt/uart: Remove '-k' option from serialcheck command
 reference

Serialcheck does not support '-k' option neither in current code or
in git history.

The code source is pointed by ddt comment at runtest/ddt/uart:2 as
git://git.breakpoint.cc/bigeasy/serialcheck.git

Signed-off-by: Orson Zhai <orson.zhai@linaro.org>
---
 testcases/ddt/scripts/uart/serialcheck.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/testcases/ddt/scripts/uart/serialcheck.sh b/testcases/ddt/scripts/uart/serialcheck.sh
index fbe284e2e..9923ba08f 100644
--- a/testcases/ddt/scripts/uart/serialcheck.sh
+++ b/testcases/ddt/scripts/uart/serialcheck.sh
@@ -53,9 +53,9 @@ run_serial_check() {
     for i in ${PORTS_TO_TEST[@]}; do    
         if [ $UART_HWFLOW -eq 0 ]; then
             echo ''; echo "Testing $i in loopback at $UART_RATE $UART_LOOPS times"
-            serialcheck -b $UART_RATE -d $i -f $temp_test_file -l $UART_LOOPS -m r -k &
+            serialcheck -b $UART_RATE -d $i -f $temp_test_file -l $UART_LOOPS -m r &
             sleep 1
-            serialcheck -b $UART_RATE -d $i -f $temp_test_file -l $UART_LOOPS -m t -k || die "TEST FAILED"
+            serialcheck -b $UART_RATE -d $i -f $temp_test_file -l $UART_LOOPS -m t || die "TEST FAILED"
         else
             echo ''; echo "Testing $i with HW flow control at $UART_RATE $UART_LOOPS times"
             serialcheck -b $UART_RATE -d $i -f $temp_test_file -l $UART_LOOPS -m r -h &
-- 
2.18.0

